utils/airtable.test.jsx

```diff
@@ -0,0 +1,38 @@
+import '@testing-library/jest-dom';
+import findMatchingRecord from './findMatchingRecord';
+
+const fakeAirtableData = [
+  {
+    _rawJson: {
+      fields: {
+        'Component Name': 'Disclosure (Show/Hide)',
+        Slug: 'disclosure',
+        'Keyboard Interactions': 'a different set of bullet points',
+        'Usage Guidelines': 'some more different paragraphs',
+      },
+    },
+  },
+];
+
+describe('findMatchingRecord', () => {
+  it('returns all of the data associated with a particular component', () => {
+    const mockComponentName = 'disclosure';
+    const matchingRecord = findMatchingRecord(
+      fakeAirtableData,
+      mockComponentName,
+    );
+    // eslint-disable-next-line no-underscore-dangle
```

Comment:
For posterity, this `eslint-disable` comment is fine because Airtable is the one imposing their dangling underscores on us.